packages/test-ovm-full-node/src/store-stress-test.ts

```diff
@@ -0,0 +1,60 @@
+/* External Imports */
+import {add0x, keccak256} from '@eth-optimism/core-utils'
+import {CHAIN_ID, GAS_LIMIT} from '@eth-optimism/ovm'
+
+import {Contract, Wallet} from 'ethers'
+import {JsonRpcProvider} from 'ethers/providers'
+import {deployContract} from 'ethereum-waffle'
+
+/* Internal Imports */
+import {getUnsignedTransactionCalldata} from './helpers'
+import {FullNodeStressTest} from './stress-test'
+
+/* Contract Imports */
+import * as SimpleStorage from '../build/SimpleStorage.json'
+
+
+class StoreStressTest extends FullNodeStressTest {
```

Comment:
Could be called `SimpleStorageStressTest` because at first glance I read this as a class which stores the results of stress tests.